driver/ppm_events.c

```diff
@@ -577,7 +587,7 @@ int val_to_ring(struct event_filler_arguments *args, uint64_t val, u32 val_len,
 			return PPM_FAILURE_BUG;
 		}
 
-#ifdef UDIG
+#if defined(UDIG) && !defined(WDIG)
```

Comment:
What are the possible combinations of these two? What do they mean?

```
!UDIG && !WDIG
!UDIG &&  WDIG
 UDIG && !WDIG
 UDIG &&  WDIG
```

tbh the nested ifdefs make the code hard to follow

Reply:
currently my approach is:
- UDIG -> common for user level instrumentation (pdig, udig, wdig...)
- WDIG -> specific for Windows

Does it make sense? Does it reflect your approach?